### DISCUSSION TITLE
How to disable background blur

### DISCUSSION THREAD

**Discussion by 0verEngineer** [Clooos/Bubble-Card#597] [General]
Since a few weeks the background is blurred when a bubble card is opened, this includes the homeassistant top bar where the hamburger menu button is, how to disable this?

**Comment by Clooos** [comment]
Hi, you can disable it here in your first pop-up settings.

![Screenshot_2024-06-24-03-52-26-166_io homeassistant companion android-edit](https://github.com/Clooos/Bubble-Card/assets/36499953/56b61c94-4b95-4146-a815-211638d1d08a)

**Reply by 0verEngineer** [reply]
I have enabled this setting on all popups and it is still the same on mobile.

**Reply by Clooos** [reply]
Have you tried to clear your cache on mobile?

**Reply by 0verEngineer** [reply]
Yes

**Reply by Clooos** [reply]
I have this toggle enabled on my main dashboard on the first pop-up and I don't have your issue so I'm clueless at the moment.

**Reply by 0verEngineer** [reply]
Now it somehow started working like it should, i don't know why but on the third time i opened the app it worked, idk why the cache and app data clear did not work...  Thank you

**Reply by Clooos** [reply]
Nice! The HA cache is really strange sometimes, glad that it's now working!

**Comment by zyi840449** [comment]
What if I don't want to cancel it, but just want to adjust the intensity?

**Reply by Clooos** [reply]
You can change the "Background/Backdrop blur" value in that same panel.
